src/imgtools/betapipeline.py

```
@@ -0,0 +1,97 @@
+    # roi_separate=False
+
+    def __post_init__(self):
+        self.crawl = Crawler()
+        self.lacer = Interlacer()
+        transforms = [Resample(self.spacing)]
+
+        # add W/L adjustment to transforms
+        if self.window is not None and self.level is not None:
+            transforms.append(WindowIntensity(self.window, self.level))
+        
+        self.loader = SampleLoader()
```

Ruff (0.8.2)

Undefined name `SampleLoader`

(F821)